File: backend/src/main/java/edu/cit/aligato/fortpointproperties/properties/dto/PropertyCardDTO.java
package edu.cit.aligato.fortpointproperties.properties.dto;

import java.util.ArrayList;
import java.util.List;

import edu.cit.aligato.fortpointproperties.properties.enums.ListingType;

public class PropertyCardDTO {
    public String id;
    public String name;
    public String basicDescription;
    public String location;
    public Double priceRangeMin;
    public Double priceRangeMax;
    public List<ListingType> listingTypes = new ArrayList<>();
    public Boolean hasPromo;
    public String coverPhotoUrl;
}

File: mobile/app/src/main/java/com/example/fortpointproperties/features/properties/data/model/PropertyCardDto.kt
package com.example.fortpointproperties.features.properties.data.model

data class PropertyCardDto(
    val id: String? = null,
    val name: String? = null,
    val basicDescription: String? = null,
    val location: String? = null,
    val priceRangeMin: Double? = null,
    val priceRangeMax: Double? = null,
    val listingTypes: List<String> = emptyList(),
    val hasPromo: Boolean? = null,
    val coverPhotoUrl: String? = null
)
